Cargo.lock

```diff
@@ -692,6 +723,16 @@ version = "2.3.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "37909eebbb50d72f9059c3b6d82c0463f2ff062c9e95845c43a6c9c0355411be"
 
+[[package]]
+name = "fixin"
+version = "0.0.1"
+source = "git+https://github.com/githedgehog/fixin?branch=main#9afddc5d845e3d7c769560be659a6cb0fbd148e2"
```

Comment:
Why is this necessary?

Reply:
The test-runner.sh launches a container which only has libc.  Proc macro crates require the rust standard library tho and this breaks our test runner.  The only easy answer was to move fixin to another repo (I can't specify the test runner on a per crate basis sadly)